src/language-css/parser-postcss.js

```diff
@@ -11,7 +11,7 @@ function parseSelector(selector) {
   if (selector.match(/\/\/|\/\*/)) {
     return {
       type: "selector-comment",
-      value: selector
+      value: selector.replace(/^ +/, "").replace(/ +$/, "")
```

Comment:
Just checking: DO you intentionally only replace spaces, or did you mean to remove other whitespace as well?

Reply:
@lydell remove only spaces, it is temporary hack because we don't parse selectors with comments, after we make sure that we do not eat comments and determine their position it will be deleted

Reply:
I see! 👍

Reply:
@lydell approve? :smile: